subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -139,152 +138,192 @@ def test_bad_modes(self, mode):
             self.fsc.validate_layout_mode(mode)
 
 
-bootloaders = [(bl, ) for bl in list(Bootloader) if bl != Bootloader.NONE]
+bootloaders_and_ptables = [(bl, pt)
+                           for bl in list(Bootloader) if bl != Bootloader.NONE
+                           for pt in ('gpt', 'msdos', 'vtoc')]
```

Comment:
Oh yikes, there is no v2 partitioning support for vtoc. That needs fixing!

Reply:
Will investigate in follow-on PR